docs/conf.py

```diff
@@ -17,7 +17,7 @@
 
 sys.path.insert(0, os.path.abspath("../"))  # NOQA
 
-import fedora_messaging
+import fedora_messaging  # NOQA
```

Comment:
We could even make it `# noqa: F401` to only ignore the unused import (assuming this is what it's about).

Reply:
It's about the import not being at the top of the file. I'll be more specific, thanks.